app/assets/stylesheets/base/colors.scss

```diff
@@ -4,6 +4,7 @@ $red:         #98002e;
 $black:       #2f2f2f;
 $warm-gray:   #edeade;
 $cool-gray:   #ecebe5;
+$law-blue-ribbon: #002f7c;
```

Comment:
Actually for this, the blue color is referenced a few times in the law theme `law.scss`, should we change all of those to use this variable?